tests/features/class-wc-tests-ces-tracks.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Customer Effort Score Survey Tests.
+ *
+ * @package Automattic\WooCommerce\Admin\Features
+ */
+
+use \Automattic\WooCommerce\Admin\Features\CustomerEffortScoreTracks;
+
+/**
+ * Class WC_Tests_CES_Tracks
+ */
+class WC_Tests_CES_Tracks extends WC_Unit_Test_Case {
```

Comment:
Thanks for including some tests!

Reply:
Sure thing! I will add more tests once we merge other PRs together.